internal/resources/responses.go

```diff
@@ -28,6 +28,12 @@ type APIError struct {
 	Message    string `json:"message"`
 }
 
+type Version struct {
+	Version string `json:"version"`
+	Major   string `json:"major"`
+	Minor   string `json:"minor"`
```

Comment:
patch is not worthy?

Reply:
For checking when to enable/disable functions patch releases can be safely ignored.

Reply:
Console version does not follow semver and some new feature (generally small features) may be added in patch versions. 

I think we can still deal with this when it occurs 🤷